.NET/Microsoft.Recognizers.Text.DataDrivenTests/Number/TestNumber_Arabic.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Microsoft.Recognizers.Text.DataDrivenTests;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Microsoft.Recognizers.Text.Number.Tests
+{
+    [TestClass]
+    public class TestNumber_Arabic : TestBase
+    {
+        public static TestResources TestResources { get; protected set; }
+
+        /*
+        [NetCoreTestDataSource]
```

Comment:
don't need them?

Reply:
Temporarily disabled, as commented in the last comit.

Reply:
Temporarily disabled, as commented in the last commit.